src/main/java/templates/mapPages.soy

```diff
@@ -0,0 +1,80 @@
+{namespace templates.pages}
+
+{template .mapPage}
+    {@param groupName : string}
+
+    <html>
```

Comment:
Why do we need the whole HTML tag here and in the main HTML file? If the main HTML file is still being used, can we just render the template into that rather than making a whole new HTML element? It feels like we shouldn't need to have multiple copies of this

    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">
    <title>The Solidarity Initiative - Event Map</title>
    <link rel="stylesheet" href="mapPage.css">
    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCm3MiL3acn2Y9ZoHXfCOWVvnJCzg2m3Us"></script>
    <script src="mapScript.js"></script>
    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">